components/Avatar/avatar.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
You can put meta tags for a description of your component library.

Reply:
ik ig we dont want this  component html to be in  SEO so it will merge with main .css  so description will be on that one
correct me if i m wrong  😅